common/api-review/generative-ai.api.md

```diff
@@ -481,7 +481,7 @@ export class GoogleGenerativeAI {
     // (undocumented)
     apiKey: string;
     getGenerativeModel(modelParams: ModelParams, requestOptions?: RequestOptions): GenerativeModel;
-    getGenerativeModelFromCachedContent(cachedContent: CachedContent, requestOptions?: RequestOptions): GenerativeModel;
+    getGenerativeModelFromCachedContent(cachedContent: CachedContent, modelParams?: Partial<ModelParams>, requestOptions?: RequestOptions): GenerativeModel;
```

Comment:
Why do we make this `Partial<>`? Is `ModelParams.model` only optional when it's passed to this method?

Reply:
None of the params are required here, and the whole object is optional, because `cachedContent` should have everything it needs to initialize (it always has "model").